python-backend/app/api/mines/notification/resources/notification.py

```diff
@@ -0,0 +1,51 @@
+from flask_restplus import Resource
+from sqlalchemy_filters import apply_sort
+
+from app.extensions import api
+from app.api.utils.include.user_info import User
+from ....utils.access_decorators import requires_role_mine_view
+from ....utils.resources_mixins import UserMixin, ErrorMixin
+from ..models.notification import Notification
+from ...mine.models.mine import Mine
+
+
+class MineNotificationResource(Resource, UserMixin, ErrorMixin):
+    @api.doc(params={'mine_guid': 'Mine guid.'})
+    @requires_role_mine_view
+    def get(self):
```

Comment:
What is this get endpoint for? probably for mine/notifications? with a few query params? we aren't exposing serial id's so there won't be direct access to this entity.

I think this will crash if you call `/mines/<mine_guid>/notifications` as there will be one too many parameters. 

Take a look at mine_party_appt_resource.py for multiple queryparams on a way to combine search params.